Makefile

```diff
@@ -1265,6 +1268,18 @@ horizon_kuttl: namespace input openstack_crds openstack_storage_crds deploy_clea
 	make cleanup
 	make infra_cleanup
 
+.PHONY: openstack_kuttl_run
+openstack_kuttl_run: ## runs kuttl tests for the openstack operator, assumes that everything needed for running the test was deployed beforehand.
+	kubectl-kuttl test --config ${OPENSTACK_KUTTL_CONF} ${OPENSTACK_KUTTL_DIR}
+
+.PHONY: openstack_kuttl
+openstack_kuttl: namespace input openstack_crds deploy_cleanup openstack openstack_deploy ## runs kuttl tests for the openstack operator. Installs openstack crds and the openstack operator, cleans up previous deployments before running the tests and, cleans up after running the tests.
```

Comment:
In this case I think we don't need `openstack_crds` as `openstack` already install the crds. Also, we probably don't need to run `openstack_deploy` as a dependency if the tests themselves apply a sample

Reply:
good points, thank you